pkg/azuredisk/controllerserver_v2.go

```diff
@@ -299,8 +299,90 @@ func (d *DriverV2) ControllerGetVolume(context.Context, *csi.ControllerGetVolume
 }
 
 // ControllerModifyVolume modify volume
-func (d *DriverV2) ControllerModifyVolume(context.Context, *csi.ControllerModifyVolumeRequest) (*csi.ControllerModifyVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+func (d *DriverV2) ControllerModifyVolume(ctx context.Context, req *csi.ControllerModifyVolumeRequest) (*csi.ControllerModifyVolumeResponse, error) {
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume ID missing in the request")
+	}
+
+	if err := d.ValidateControllerServiceRequest(csi.ControllerServiceCapability_RPC_MODIFY_VOLUME); err != nil {
+		return nil, status.Errorf(codes.Internal, "invalid modify volume req: %v", req)
+	}
+	diskURI := volumeID
+
+	if err := azureutils.IsValidDiskURI(diskURI); err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "disk URI(%s) is not valid: %v", diskURI, err)
+	}
+
+	diskName, err := azureutils.GetDiskName(diskURI)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, err.Error())
+	}
+
+	if _, err := d.checkDiskExists(ctx, diskURI); err != nil {
+		return nil, status.Error(codes.NotFound, fmt.Sprintf("Volume not found, failed with error: %v", err))
+	}
+
+	mutableParams := req.GetMutableParameters()
```

Comment:
remove this `mutableParams` var define, just use `req.GetMutableParameters()` in L327

Reply:
remove it